MekHQ/src/mekhq/Utilities.java

```diff
@@ -883,18 +904,65 @@ else if(u.getEntity() instanceof VTOL) {
         if (null != consoleCmdr) {
             result.put(CrewType.TECH_OFFICER, Collections.singletonList(consoleCmdr));
         }
+        //endregion Data Gathering
         return result;
     }
 
+    /**
+     * Function that determines what name should be used by a person that is created through crew
+     * And then assigns them a pre-selected portrait, provided one is to their index
+     * Additionally, any extraData parameters should be migrated here
+     * @param p           the person to be renamed, if applicable
+     * @param oldCrew     the crew object they were a part of
+     * @param crewIndex   the index of the person in the crew
+     * @param crewOptions whether or not to run the populateOptionsFromCrew for this person
+     */
+    private static void migrateCrewData(Person p, Crew oldCrew, int crewIndex, boolean crewOptions) {
```

Comment:
Is this temporary? (given the name `migrate...`)

Reply:
It is not, it is used to migrate crew data from MegaMek to MekHQ